app/controllers/wiki_controller.rb

```diff
@@ -341,7 +341,7 @@ def index
       .references(:node_revisions)
       .group('node_revisions.nid, node_revisions.vid')
       .order(order_string)
-      .where("node_revisions.status = 1 AND node.status = 1 AND type = 'page'"))
+      .where("node_revisions.status = 1 AND node.status = 1 AND (type = 'page' OR type = 'tool' OR type = 'place')"), items: 10)
```

Comment:
Hi @gaurav2699, thanks for opening this up.

I know that `type = 'page'` represents wikis, and I think that this query before was meant to display wikis only, why do we need to add `type = tool OR type = place`? Are they related to the wiki pages?